safe/definitions/fields.py

```diff
@@ -1448,6 +1448,36 @@
     'default_value': None
 }
 
+# Fatality ratio
+population_fatality_ratio_field = {
+    'key': 'fatality_ratio_field',
+    'name': tr('Fatality Ratio'),
+    'field_name': 'fatality_ratio',
+    'type': QVariant.Double,
+    'length': default_field_length,
+    'precision': 10,  # I think we need some precision for this field.
+    'absolute': False,
+    'description': tr(
+        'The population fatality ratio for a given hazard class.'),
+    'help_text': tr(
+        '"Killed" is defined as: {concept} In cases where population data '
```

Comment:
The field name is `Fatality Ratio`, but the help text or description doesn't explain about it. It explains about `killed` people.